.gitignore

```diff
@@ -60,3 +60,5 @@ config/certs
 
 dump.rdb
 /.project
+.generators
+.rakeTasks
```

Comment:
Seems like this is probably unrelated to this PR?

Reply:
some files that intellij creates, was adding them in so they are ignored